### ISSUE TITLE
Make content element spacing more consistent

### ISSUE THREAD

**Issue by danfox01** [baltimorecounty/BCPL-assets#496]
There's a little bit of variation in the bottom/top margins of some of our elements -- H2s, H3s, Ps, ULs, accordions, etc. -- that is causing some pages to look a little choppy. This is a good example: https://www.bcpl.info/youth/teachers.html.

@martypowell and @sdurphy can we talk about some spacing tweaks to make the groupings more consistent? We might need to set up a demo page in dev with a bunch of different element combinations to figure out what works.

**Comment by martypowell** [comment]
I think a demo page would be good, that sam can decide. I'll offer any input if I have any.

**Comment by martypowell** [comment]
@danfox01 assigning to @sdurphy for a demo page.

**Comment by danfox01** [comment]
Threw this together so Sam doesn't have to: https://www.bcpl.info/_test/copy-style-test.html

A lot of these are edge cases, but show a lot of combinations that result in weird spacing.

**Comment by martypowell** [comment]
@danfox01 great, now should this be assigned to sam, to determine the spacing rules?

**Comment by danfox01** [comment]
@martypowell, yeah I'd like @sdurphy to look at it. It's still assigned to him.

**Comment by martypowell** [comment]
waiting for sam to grab this

**Comment by sdurphy** [comment]
Can we make the following changes for this in dev?:

1. For "h1, h2, h3" can we make the margin-top ,and margin-bottom, 25px?
2. Can we remove "margin:o;" from ".address span + .content-accordion, .address strong + .content-accordion, .ekko-lightbox.modal .modal-title + .content-accordion, p + .content-accordion"
3. For ".btn, .buttonPrimary, .seButton, .seButton.buttonPrimary" can we change the margin-bottom from 0 to 30px?

**Comment by martypowell** [comment]
I can hop on this today.

**Comment by martypowell** [comment]
@sdurphy can please provide sample links for each of the selectors in item number 2?

**Comment by sdurphy** [comment]
https://www.bcpl.info/services/meeting-rooms-study-rooms.html#meetingroompolicy

https://www.bcpl.info/youth/birth-to-five.html#BabyBoosters

**Comment by martypowell** [comment]
ok, I think maybe i didn't understand how sass was creating the sass file. Can you confirm in dev that all spacing issues have been resolved?
